docs/4.0/components/forms.md

```diff
@@ -1118,6 +1118,32 @@ Custom checkboxes and radios can also be disabled. Add the `disabled` boolean at
 </div>
 {% endexample %}
 
+### Switches
+
+A switch has the markup of a custom checkbox but uses the `.custom-switch` class to render a toggle switch. Switches also support the `disabled` attribute.
+
+{% example html %}
+<div class="custom-control custom-switch">
+  <input type="checkbox" class="custom-control-input" id="customSwitch1">
+  <label class="custom-control-label" for="customSwitch1">Toggle this switch element</label>
+</div>
+<div class="custom-control custom-switch">
+  <input type="checkbox" class="custom-control-input" disabled id="customSwitch2">
+  <label class="custom-control-label" for="customSwitch2">Disabled switch element</label>
+</div>
+{% endexample %}
+
+#### Animated switch
+
+Optionally you can add the class `.custom-switch-animated` to add a transition to the indicator.
+
+{% example html %}
+<div class="custom-control custom-switch custom-switch-animated">
```

Comment:
`.custom-switch-animated` is not defined.

Reply:
have a look at https://github.com/twbs/bootstrap/pull/23004/files#diff-e5540857a54676b43146ad92add2dbdeR182
it's actually is defined as a nested sass selector

Reply:
They don't like that kind of stuff. Don't use nesting to create new classes